packages/SketchMorph2-Core.package/MorphicMonet.class/instance/chooseColor.st

```diff
@@ -0,0 +1,9 @@
+tooling
+chooseColor
+	| currentColor |
+	self world activeHand showTemporaryCursor: 
+		(CursorWithAlpha fromIcon: UiFugueIcons pipetteIcon).
+	[Sensor redButtonPressed] whileFalse:
```

Comment:
I do not like this solution at all. It
- doesn't allow the pipette to be tested.
- uses up one CPU thread. These things don't grow on trees.
- stops the execution of everything else in Squeak. I do believe that there could be 100 things that break that we just don't notice.
- creates artifacts when resizing Squeak while the pipette is active: 
![image](https://user-images.githubusercontent.com/68013019/118792643-2463eb80-b898-11eb-95bf-0baff1de9531.png)

Note that this list may not be complete.

What about snapshotting the world and displaying it until a mouse down/up event gets triggered?

Reply:
If we follow the snapshot approach, I'd prefer the invisible morph technique since it does basically the same and doesn't "freeze" the frame like a snapshot would.

Reply:
I'd be fine with this too. But I think, the other one would be more like other software. When you make a screenshot with e.g. SnippingTool, it also freezes the background. But as long as we don't have this loop, I'm happy.